frontend/views/containers/contributions/IncomeDetails.vue

```diff
@@ -0,0 +1,239 @@
+<template lang='pug'>
+modal-base-template(ref='modal')
+  .wrapper-container
+    .c-content
+      i18n.c-title(tag='h2') Income Details
+
+      form.card.c-card(
+        @submit.prevent='submit'
+        novalidate='true'
+      )
+        fieldset.c-firstQuestion(v-error:incomeDetailsKey='{position: "beforeend"}')
+          legend.label
+            | {{ L('Do you make at least {groupMincomeFormatted} per month?', { groupMincomeFormatted }) }}
+            tooltip(:text='L("This is the minimum income in your group")' direction='top')
+              button.is-icon-smaller.is-primary.c-tip(:aria-label='L("This is the minimum income in your group")')
+                i.icon-info
+          label.radio
+            input.input(
+              type='radio'
+              name='incomeDetailsKey'
+              value='pledgeAmount'
+              v-model='$v.form.incomeDetailsKey.$model'
+              @change='resetAmount'
```

Comment:
This `resetAmount` thing wasn't needed when I implemented, and I don't see the reason for it here...

Reply:
So, when we change the option (pledge vs receive), it fills correctly the amount input and removes any existing error if avalable.